spec/requests/admin/users_requests_spec.rb

```diff
@@ -237,6 +237,17 @@
         end
       end
     end
+    describe "Validate Order of returned organization list" do
+      let!(:pawnee_organization) { create(:organization, id: 2, name: "Pawnee") }
+      let!(:sf_diaper_organization) { create(:organization, id: 3, name: "SF Diaper") }
+      let!(:second_city_organization) { create(:organization, id: 4, name: "Second City") }
```

Comment:
Can you move these to a `before(:each)` block? Since we don't reference this data in the test, we don't need to assign it names.